workbench-core/example/app/src/routes/dynamicAuthorizationRoutes.ts

```diff
@@ -45,4 +48,24 @@ export function setUpDynamicAuthorizationRoutes(router: Router, service: Dynamic
       }
     })
   );
+  router.post(
+    '/authorization/identitypermissions',
+    wrapAsync(async (req: Request, res: Response) => {
+      try {
+        const params = req.body;
```

Comment:
should we be validating that the request body has the variables and types we expect?

Reply:
I do agree with this but we aren't testing the service at this point. Just the implementation of our example application.

Reply:
Even if its an example app, we should be following best practices and sanitizing inputs.

Reply:
That is correct